Microsoft.Toolkit.Uwp.UI.Animations/Xaml/Abstract/Animation.cs

```diff
@@ -91,14 +91,14 @@ public abstract class Animation : DependencyObject, ITimeline
         /// </summary>
         public RepeatOption Repeat
         {
-            get => (RepeatOption)GetValue(RepeatOptionProperty);
-            set => SetValue(RepeatOptionProperty, value);
+            get => (RepeatOption)GetValue(RepeatProperty);
+            set => SetValue(RepeatProperty, value);
         }
 
         /// <summary>
         /// Identifies the <seealso cref="Repeat"/> dependency property.
         /// </summary>
-        public static readonly DependencyProperty RepeatOptionProperty = DependencyProperty.Register(
+        public static readonly DependencyProperty RepeatProperty = DependencyProperty.Register(
```

Comment:
I know this is a 'breaking' change, but this is broken if it's not fixed, eh?

We're going to do a 7.0.1 patch soon, so wondering if we should take this whole PR in?

Reply:
It should technically not be broken, as I don't think dependency properties have a strict requirement to be named like their accessor properties in order to work (unlike with attached properties, where that is the case). So barring some weird hardcoding in the XAML compiler, it should still work (and it does in the sample app as far as I can tell). That said, I think it would make sense to slip this fix into 7.0.1 if we can though, so that it's out as soon as possible.

Also to be clear, this would be binary breaking, but not source breaking (assuming you're not accessing the dependency property directly, which you shouldn't be doing anyway). Consumers updating to a hypothetical 7.0.1 version with this fix shouldn't notice the difference.